test/jdk/java/net/DatagramSocket/InterruptibleDatagramSocket.java

```diff
@@ -95,12 +98,17 @@ else if (!s.isClosed() && interruptible)
 
     public static void main(String[] args) throws Exception {
         try (DatagramSocket s = new DatagramSocket()) {
+            System.out.println("Established uninterruptible datagram socket "
```

Comment:
"Established uninterruptible datagram socket" is a bit strange. I think I you want to this to say "Created datagram socket bound to port %d".

Reply:
I think information on the scenario under test would be useful, such as, 
"Testing interrupt of DatagramSocket  receive on endpoint " + s.getLocalSocketAddress()

"Testing interrupt of MulticastSocket receive on endpoint " + s.getLocalSocketAddress()

"Testing interrupt of DatagramChannel socket receive on endpoint " + s.getLocalSocketAddress()

Reply:
Sure, I can change that.